src/modules/login/components/MessageBox.js

```diff
@@ -13,9 +12,9 @@ class MessageBox extends Component {
         if (this.props.errorMessage != null || this.props.errorMessage != undefined) {
```

Comment:
This is a redundant check. Your checking if the value is undefined and null, and although they're not entirely the same, in this case it is.